website/build.gradle

```diff
@@ -0,0 +1,54 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * License); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an AS IS BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// Define common lifecycle tasks and artifact types
+apply plugin: "base"
+
+task cleanWebsite(type: Delete) {
+  delete '.sass-cache', '.testcontent', 'content'
+}
+clean.dependsOn cleanWebsite
+
+task installBundler(type: Exec) {
+  commandLine './install_bundler.sh'
```

Comment:
Perhaps `commandLine` is a special case, but I believe the way you wrote the tasks in this file makes them execute during Gradle's configuration phase (regardless of the task chosen to be executed). I would put the commands in these tasks within a `doLast` closure.

Here's an example from the documentation:
```
task testBoth {
    doFirst {
      println 'This is executed first during the execution phase.'
    }
    doLast {
      println 'This is executed last during the execution phase.'
    }
    println 'This is executed during the configuration phase as well.'
}
```
https://docs.gradle.org/current/userguide/build_lifecycle.html

Reply:
I believe the syntax you suggest is more powerful, but for this simple command they are equivalent. Both syntaxes are shown in the first example on [Authoring Tasks: Defining Tasks](https://docs.gradle.org/current/userguide/more_about_tasks.html#sec:defining_tasks)

The syntax here is registering a new task named `installBundler` of type `Exec`. The configuration within it is specific to the [`Exec` task type](https://docs.gradle.org/current/dsl/org.gradle.api.tasks.Exec.html).